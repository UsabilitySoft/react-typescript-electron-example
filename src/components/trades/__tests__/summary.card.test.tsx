import React from 'react';
import renderer from 'react-test-renderer';

import { SummaryCard } from '../summary.card';

it('renders correctly when there are no items', () => {
  const tree = renderer.create(<SummaryCard />).toJSON();
  expect(tree).toMatchSnapshot();
});