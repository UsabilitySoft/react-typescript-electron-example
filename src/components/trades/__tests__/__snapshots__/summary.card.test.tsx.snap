// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    className="MuiCardContent-root"
  >
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
    />
    <h2
      className="MuiTypography-root MuiTypography-h5"
    />
  </div>
</div>
`;
